cache/src/main/java/io/envoyproxy/controlplane/cache/V2SimpleCache.java

```diff
@@ -0,0 +1,9 @@
+package io.envoyproxy.controlplane.cache;
+
+import static io.envoyproxy.controlplane.cache.Resources.V2_TYPE_URLS;
+
+public class V2SimpleCache<T> extends SimpleCache<T, V2Snapshot> {
```

Comment:
These classes make it so callers don't have to worry about the new type variable on SimpleCache, and also passing type URLs when instantiating the cache